t/50chopblanks.t

```diff
@@ -26,42 +26,59 @@ ok $dbh->do("DROP TABLE IF EXISTS dbd_mysql_t50chopblanks"), "drop table if exis
 my $create= <<EOT;
 CREATE TABLE dbd_mysql_t50chopblanks (
   id INT(4),
-  name TEXT
+  c_varchar VARCHAR(10),
```

Comment:
I've removed the VARCHAR and VARBINARY from the test case.
What I need to test is that binary fields are not being trimmed on the client side, and that's accomplished by BLOB and it's variants.